libr/core/rvc.c

```diff
@@ -654,6 +654,49 @@ static RList *blobs_add(const char *rp, const RList *files) {
 	return NULL;
 }
 
+R_API char *r_vc_find_rp(const char *path) {
+	{
+		int ret = repo_exists (path);
+		switch (ret) {
+		case 0:
+			break;
+		case 1:
+			return r_str_new (path);
+		case -1:
+			printf ("A corrupted repo may have been found at %s, please refrain from naming any files .rvc\n", path);
```

Comment:
Its not resolved . Use eprintf and add a break

Reply:
Sorry about that mixed it up with the one above